starter-code/scripts/article.js

```diff
@@ -6,31 +6,45 @@ function Article (opts) {
   // TODO: Use the JS object passed in to complete this constructor function:
   // Save ALL the properties of `opts` into `this`
   // lots of this.someProperty = opts.property
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
 }
 
+
+
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
   /* TODO: This cloned article is no longer a template,
   as it now has real data attached to it! We need to account
   for that before this current article gets rendered to our
   DOM. */
   // one line of code to remove the class that hides the template so that the articles will actually show
+  $newArticle.removeClass('template');
 
   if (!this.publishedOn) $newArticle.addClass('draft');
   $newArticle.data('category', this.category);
 
   /* TODO: Now use jQuery to fill in the rest of the current
   template clone with properties from this particular Article instance.
-  We need to fill in:
-    1. author name,
-    2. author url,
-    3. article title,
-    4. article body, and
-    5. publication date. */
+  We need to fill in:*/
+$newArticle.find('a').html(this.author);
```

Comment:
So good start so far. With the other properties, its key to focus on what element you are targeting with jQuery, and what exactly are you trying to manipulate. So you should be looking back and forth from this current file to your index.html. Now that you two have worked through lab 3, you should have a better understanding of whats going on/needs to be completed.